### ISSUE TITLE
Uncaught TypeError: Cannot read property 'cleanup' of null

### ISSUE THREAD

**Issue by xlc** [mozilla/pdf.js#6440]
Chrome 46 using demo viewer

Open sidebar without a document loaded will throw following exception:
You can "unload" the file in demo viewer by open a non-pdf file.
```
Uncaught TypeError: Cannot read property 'cleanup' of null
pdfViewCleanup @ viewer.js:6863
setTimeout (async)PDFRenderingQueue_renderHighestPriority @ viewer.js:3407
pdfViewForceRendering @ viewer.js:6869
(anonymous function) @ viewer.js:7202
```

**Comment by Snuffleupagus** [comment]
This might be relevant here: https://github.com/mozilla/pdf.js/pull/5633#issuecomment-69432198.

**Comment by Snuffleupagus** [comment]
Closing, since this should fixed by PR #6546.
